pkg/proxy/proxy.go

```diff
@@ -75,6 +77,11 @@ func (pm *Manager) cacheCleanup(ctx context.Context) {
 
 func getTokenHash(req *http.Request) (string, error) {
 	c, err := req.Cookie(ObotAccessTokenCookie)
+	if errors.Is(err, http.ErrNoCookie) {
+		// Check the zero cookie. This one is present when the token is too large to fit in one cookie and
+		// must be split into two.
+		c, err = req.Cookie(ObotAccessTokenCookieZero)
+	}
```

Comment:
This seems a little strange to me, so I just want to double-check: we want to look for the `_0` cookie only if the non-`_0` cookie doesn't exist? If we have split the cookie into two parts, then I would expect that both cookies should exist.

Reply:
Yes, this is intentional. If the token is in one cookie, it will be called `obot_access_token`. If it is in more than one cookie, it will be in `obot_access_token_0`, `obot_access_token_1`, and so on. So the presence of `_0` indicates a multi-part token. In this case we only bother to hash the first piece of it, since there's really no need to hash the whole thing, as all we need is a unique identifier.

Reply:
^ Let me know if you think any of that info should go into the code comments.